test/TensorFlowRuntime/random.swift

```diff
@@ -0,0 +1,183 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+//
+// TensorFlow Random library tests
+
+import StdlibUnittest
+import TensorFlow
+
+#if TPU
+import TensorFlowUnittestTPU
+#else
+import TensorFlowUnittest
+#endif
+import StdlibUnittest
+
+var RandomTests = TestSuite("Random")
+
+RandomTests.test("UInt8ArrayInitializers") {
+  let _ = ARC4RandomEngine(seed: [0])
+  let _ = ARC4RandomEngine(seed: [1, 2, 3, 4, 5, 6, 7])
+  let _ = ARC4RandomEngine(seed: Array(repeating: UInt8.max, count: 256))
+}
+
+RandomTests.test("BinaryIntegerInitializers") {
+  let _ = ARC4RandomEngine(seed: UInt32(0))
+  let _ = ARC4RandomEngine(seed: UInt64(1))
+  let _ = ARC4RandomEngine(seed: Int16(-42))
+}
+
+RandomTests.test("TestVector1") {
+  let engine = ARC4RandomEngine(seed: [1, 2, 3, 4, 5, 6, 7, 8])
+  let expected: [UInt8] = [0x97, 0xab, 0x8a, 0x1b, 0xf0, 0xaf, 0xb9, 0x61,
+                           0x32, 0xf2, 0xf6, 0x72, 0x58, 0xda, 0x15, 0xa8,
+                           0x82, 0x63, 0xef, 0xdb, 0x45, 0xc4, 0xa1, 0x86,
+                           0x84, 0xef, 0x87, 0xe6, 0xb1, 0x9e, 0x5b, 0x09]
+  let actual: [UInt8] = engine.generate(count: 32)
+  expectEqual(actual, expected)
+}
+
+RandomTests.test("TestVector2") {
+  let engine = ARC4RandomEngine(seed: [0x1a, 0xda, 0x31, 0xd5, 0xcf, 0x68, 0x82, 0x21,
+                                       0xc1, 0x09, 0x16, 0x39, 0x08, 0xeb, 0xe5, 0x1d,
+                                       0xeb, 0xb4, 0x62, 0x27, 0xc6, 0xcc, 0x8b, 0x37,
+                                       0x64, 0x19, 0x10, 0x83, 0x32, 0x22, 0x77, 0x2a])
+  let expected: [UInt8] = [0x37, 0x0b, 0x1c, 0x1f, 0xe6, 0x55, 0x91, 0x6d,
+                           0x97, 0xfd, 0x0d, 0x47, 0xca, 0x1d, 0x72, 0xb8]
+  let _ = engine.generate(count: 0x1000)
+  let actual: [UInt8] = engine.generate(count: 16)
+  expectEqual(actual, expected)
+}
+
+RandomTests.test("CopyDoesntBreakOriginal") {
+  let engine1 = ARC4RandomEngine(seed: [0x1a, 0xda, 0x31, 0xd5, 0xcf, 0x68, 0x82, 0x21,
+                                       0xc1, 0x09, 0x16, 0x39, 0x08, 0xeb, 0xe5, 0x1d,
```

Comment:
Please indent these three lines (beginning with `0xc1`, `0xeb`, `0x64`) with an extra space.
Same for the `CopyIsDistinctFromOriginal` test below.

Reply:
Reformatted.